docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,105 @@
+# VisualDL Web App Develop
+
+VisualDL has two major components. One is the Python/C++ SDK that logs the matrix during training,
```

Comment:
Are you sure the SDK logs the matrices? If I am not wrong, they record vectors or scalars only

Reply:
@varunarora matrices is my fancy word for data. I can change if it's inappropriate.

Reply:
Haha unfortunately, matrices is a very loaded word. When one says "matrices", they are usually implying multidimensional data. Unfortunately, VDL doesn't do anything with multidimensional data today.